generate/generate.go

```diff
@@ -21,7 +21,8 @@ var (
 
 // Generator represents a generator for a container spec.
 type Generator struct {
-	spec *rspec.Spec
+	spec         *rspec.Spec
+	hostSpecific bool
```

Comment:
On Fri, Jul 22, 2016 at 10:16:03AM -0700, hmeng-19 wrote:
> > @@ -21,7 +21,8 @@ var (
> >  
> >  // Generator represents a generator for a container spec.
> >  type Generator struct {
> > -	spec *rspec.Spec
> > +	spec         *rspec.Spec
> > +	hostSpecific bool
> 
> I currently did not think of any cases where
> `Generator.hostSpsecific` is queried outside the package.

“I can't think of any reason you'd care” is different from “we don't
want people touching this directly”.  Since a public field doesn't
require getters/setters, I'd rather keep things public unless we have
reasons to make them private.